src/nodes/mono_camera_node.cpp

```diff
@@ -1,15 +1,26 @@
 #include <ros/ros.h>
 #include <avt_vimba_camera/mono_camera.h>
+#include <avt_vimba_camera/status_camera.h>
 
 int main(int argc, char** argv)
 {
-	ros::init(argc, argv, "mono_camera_node");
-
+  ros::init(argc, argv, "mono_camera_node");
   ros::NodeHandle nh;
   ros::NodeHandle nhp("~");
-
+  ros::Rate loop_rate(10);
+  Status_camera h(nh,nhp);
+  h.pre_camera();
   avt_vimba_camera::MonoCamera mc(nh,nhp);
+  h.post_camera();
 
-  ros::spin();
+while(ros::ok())
+{
+
+h.status1=mc.flag;
+mc.time_compare();
```

Comment:
You may want to do the status updates after spinOnce. Otherwise, you will miss one spin when you actually do the comparison.